include/net/ethernet.h

```diff
@@ -379,6 +396,15 @@ struct ethernet_context {
 	int8_t vlan_enabled;
 #endif
 
+#if defined(CONFIG_NET_DSA)
```

Comment:
Put it above CONFIG_NET_VLAN and reverse the order of the attributes you added, for better alignment.

Reply:
Ok.